mypy/stubutil.py

```diff
@@ -155,8 +157,11 @@ def generate_guarded(mod: str, target: str,
             print('Created %s' % target)
 
 
-def report_missing(mod: str) -> None:
-    print('Failed to import {}; skipping it'.format(mod))
+def report_missing(mod: str, message: Optional[str] = None) -> None:
+    if message:
+        print('Failed to import {} with error: {}; skipping it'.format(mod, message))
+    else:
+        print('Failed to import {}; skipping it'.format(mod))
```

Comment:
I would avoid the duplication by making the default value for `message` an empty string (in the function signature) and writing this as:
```python
if message:
    message = ' with error: ' + message
print('Failed to import {}{}; skipping it'.format(mod, message))
```